project/assets/login.less

```diff
@@ -0,0 +1,90 @@
+.body-structure {
+    width: 100%;
+}
+
+.body-skin {
+    background-color: rgb(230, 230, 230);
+}
+
+header {
+    background: url("images/feedback-banner.jpg") center center no-repeat;
+    opacity: 0.8;
+    height: 600px;
+    width: 100%;
+
+    .grid {
+        height: 100%;
+    }
+}
+
+.login-section-1 {
+    padding: 70px;
+}
+
+.login-section-1-skin {
+    background-color: rgb(255, 255, 255);
+}
+
+.login-section-text {
+    padding: 70px;
+}
+
+.login-section-2 {
+    padding: 30px;
+}
+
+.login-form {
+    align-self: center;
+    padding-bottom: 30px;
+    margin-left: 20%;
+    width: 25%;
+    height: 65%;
+}
+
+.login-form-skin {
+    background-color: rgb(255, 255, 255);
```

Comment:
login form skin and login section skin are identical. Find a more generic name and use them in both places.